server/src/modules/users/user.entity.ts

```diff
@@ -38,8 +39,8 @@ export class UserEntity extends CommonEntity {
   email: string;
 
   @ApiProperty()
-  @Column()
-  role: number;
+  @Column({ default: UserRoleEnum.USER })
```

Comment:
add enum: UserRoleEnum option, since this is enum field, also add nullable: false